src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import BreadcrumbYearNavigation from './BreadcrumbYearNavigation';
+import type { NavigationBreadCrumb } from '@ses/components/Breadcrumbs/Breadcrumbs';
```

Comment:
The correct naming would be `NavigationBreadcrumb` (without the capital C)

Reply:
For this , you will see more files change, but just correct the name in all of them
Done